03.asynchronous/callback.js

```diff
@@ -0,0 +1,22 @@
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  function () {
+    db.run("INSERT INTO books(title) VALUES (?)", "JavaScript", function () {
```

Comment:
CREATE TABLE とスタイルが揃っていなさそうです。

```suggestion
    db.run("INSERT INTO books (title) VALUES (?)", "JavaScript", function () {
```